### PR TITLE
Femtomes user guide

### PR DESCRIPTION
**Pull request by Femtomes**
Hi @hamishwillee. 
The user guide file is introduce for femtomes gps driver. we hope to merge in there.
Thank you! !

### CODE REVIEW

File: en/gps_compass/rtk_gps_fem_mini2.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+COMPACT, DUAL ANTENNA, DUAL FREQUENCY GNSS OEM BOARD DELIVERS CENTIMETER ACCURATE POSITIONS AND PRECISE HEADING
+
+<img src="http://www.femtomes.com/image/PX4/FB_1.jpg" width="400px" title="FB672" />
+
+[Built in FB6A0](http://www.femtomes.com)
+
+[FB6A0](http://www.femtomes.com) High precision positioning
+COMPACT, QUADRUPLE FREQUENCY GNSS OEM BOARD DELIVERS CENTIMETER ACCURATE POSITIONS
+
+<img src="http://www.femtomes.com/image/PX4/FB_2.jpg" width="400px" title="FB6A0" />
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps_fem_mini2.md:59]: normally we'd pull in these images into the build. That way if your site goes down we still have them locally. I can do that as a post process.

File: en/gps_compass/rtk_gps_fem_mini2.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<img src="http://www.femtomes.com/image/PX4/MINI_II_2.jpg" width="300px" title="port" />
+
+## Configuration
+
+For heading estimation the two antennas need to be on the same level and at least 30 cm apart from each other.
+The direction that they are facing does not matter as it can be configured with the [GPS_YAW_OFFSET](../advanced_config/parameter_reference.md#GPS_YAW_OFFSET) parameter.
+
+Configure the serial port on which the [MINI2 Receiver](http://www.femtomes.com)  will run using [GPS_1_CONFIG](../advanced_config/parameter_reference.md#GPS_1_CONFIG), and set the baud rate to 115200 using [SER_GPS1_BAUD](../advanced_config/parameter_reference.md#SER_GPS1_BAUD).
+
+[Built in FB672](http://www.femtomes.com)
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps_fem_mini2.md:47]: Everything below here does not look like it is part of a section called "Configuration". Is this some kind of additional specification?
Perhaps put a heading above here. 

```
## Additional Information

The MINI2 incorporates the following components.

File: en/gps_compass/rtk_gps_fem_mini2.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+[FB672](http://www.femtomes.com) High-precision Heading 
+COMPACT, DUAL ANTENNA, DUAL FREQUENCY GNSS OEM BOARD DELIVERS CENTIMETER ACCURATE POSITIONS AND PRECISE HEADING
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps_fem_mini2.md:50]: None of this should be capitalised.

File: en/gps_compass/rtk_gps_fem_mini2.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+<img src="http://www.femtomes.com/image/PX4/FB_1.jpg" width="400px" title="FB672" />
+
+[Built in FB6A0](http://www.femtomes.com)
+
+[FB6A0](http://www.femtomes.com) High precision positioning
+COMPACT, QUADRUPLE FREQUENCY GNSS OEM BOARD DELIVERS CENTIMETER ACCURATE POSITIONS
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps_fem_mini2.md:57]: This should not be captilalised. Also add commas between terms.

File: en/gps_compass/rtk_gps_fem_mini2.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+- 5Hz 10Hz 20Hz 
+- INS 
+- HEADING
+- OBS
+- RTK
+- BASE 
+
+## Purchase
+
+* [http://www.femtomes.com](http://www.femtomes.com)
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps_fem_mini2.md:24]: Would I be correct that all purchase are actually by contacting sales? I.e. there is  no online "shop"?

File: en/gps_compass/rtk_gps_fem_mini2.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+* [sales@femtomes.com](mailto:sales@femtomes.com)
+* TEL: +86-10-53779838
+
+## Functional Ports
+
+<img src="http://www.femtomes.com/image/PX4/MINI_II_1.jpg" width="500px" title="mini2port" />
+
+## Wiring and Connections
+
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps_fem_mini2.md:33]: Above you say "Customers benefit from the Ethernet connectivity available on the board, allowing high speed data transfer" but this only covers connecting via UART. Nor CAN.

1. Do you provide any cables to make it easy to connect? If so, what are they?
2. It is useful to create a little table like this one to make the wiring very easy for usrs: http://docs.px4.io/master/en/gps_compass/rtk_gps_cuav_c-rtk.html#pinout I would normally do it with Pixhawk 4 as the flight controller.
3. What have you tested this for with PX4? Ie has this been tested on CAN vs PX4 running on what hardware. Same with Ethernet. If you used Ethernet what hardware and how was it set up?

**Comment by hamishwillee** [comment]
To make this build we would need to add an entry to the /en/SUMMARY.md doc.

**Comment by hamishwillee** [comment]
Thanks for this, some comments above.

File: en/gps_compass/rtk_gps_fem_mini2.md (hunk 6 above)
Comment (bys1123) [reply en/gps_compass/rtk_gps_fem_mini2.md:33]: The Ethernet connectivity is for PC I suppose.

Comment (hamishwillee) [reply en/gps_compass/rtk_gps_fem_mini2.md:33]: > The Ethernet connectivity is for PC I suppose.

The docs should state how this works and what has been tested on PX4, not "generically". So something like "For PX4 you can connect using serial and X, Y ,Z. Ethernet can be used to configure the RTK from a PC (via a web interface).

Does that make sense?

File: en/gps_compass/rtk_gps_fem_mini2.md (hunk 5 above)
Comment (Femtomes) [reply en/gps_compass/rtk_gps_fem_mini2.md:24]: yes, we have no online "shop"

**Comment by hamishwillee** [comment]
Hi @Femtomes 

Did you see my comments/are you still working on this? How can I help you progress?

File: en/gps_compass/rtk_gps_fem_mini2.md (hunk 1 above)
Comment (Femtomes) [reply en/gps_compass/rtk_gps_fem_mini2.md]: hello, we have upload pictures in this commit

**Comment by hamishwillee** [comment]
@Femtomes I have subedited this to make it a little more readable. Are you OK for me to merge as-is?

Can you confirm that the instructions here: http://docs.px4.io/master/en/gps_compass/rtk_gps.html also apply to how this is used? [ie that the whole survey-in process is the same?]

**Comment by Femtomes** [comment]
> @Femtomes I have subedited this to make it a little more readable. Are you OK for me to merge as-is?
> 
> Can you confirm that the instructions here: http://docs.px4.io/master/en/gps_compass/rtk_gps.html also apply to how this is used? [ie that the whole survey-in process is the same?]

@hamishwillee thanks so much.
1、you can merge with your lastest fix. 
2、the whole survey-in process is the same

**Comment by hamishwillee** [comment]
Awesome! I've merged and done a post-process fix in https://github.com/PX4/px4_user_guide/pull/768

If you look at that, the missing thing is the change to SUMMARY.md. Without that the gitbook does not create HTML for the markdown page.

**Pull request merged**